Essentials/src/messages_en.properties

```diff
@@ -601,8 +601,6 @@ versionOutputUnsupported=§d{0} §6version\: §d{1}
 versionOutputUnsupportedPlugins=§6You are running §dunsupported plugins§6\!
 versionMismatch=§4Version mismatch\! Please update {0} to the same version.
 versionMismatchAll=§4Version mismatch\! Please update all Essentials jars to the same version.
-voiceSilenced=§6Your voice has been silenced\!
-voiceSilencedReason=§6Your voice has been silenced\! Reason\: §c{0}
```

Comment:
These appear to have been inadvertently removed

Reply:
Intentional